src/integrationTest/resources/application-integration.yaml

```diff
@@ -60,6 +60,9 @@ service-config:
         - Bulk_Scanned
       form-type-to-surname-ocr-field-mappings:
         - formType: form1
-          ocrField: lastName
+          ocrFieldList :
```

Comment:
`ocrFields` should be enough. We know it's a list. What do you think?

P.S. There is no space before `:` ;)